### ISSUE TITLE
Joystick deadzone configuration 

### ISSUE THREAD

**Issue by jamesadevine** [microsoft/jacdac#592]
Fixing joystick deadzone is required before we can produce more joystick modules.

**Comment by pelikhan** [comment]
If needed, filter out deadzone in makecode/ts

**Comment by pelikhan** [comment]
Easy fix?

**Comment by jamesadevine** [comment]
yes, that would work for now, but the long term fix is to add a deadzone in joystick firmware.

**Comment by pelikhan** [comment]
> but the long term fix is to add a deadzone in joystick firmware.

Yes that's what i meant. We should do deadzone in the firmware.

**Comment by pelikhan** [comment]
@mmoskal firmware should do deadzone detection.

**Comment by jamesadevine** [comment]
This is not for hackathon, and it is not a priority.

**Comment by jamesadevine** [comment]
(the reason being that we have already programmed PADAUKs and put them on PCBs. Based on the assumption that we are shipping kits this week).

Current modules will still be able to be used (even if the deadzone is introduced).

**Comment by mmoskal** [comment]
This was definitely not done. We should at least spec it.
